DirectProgramming/C++SYCL_FPGA/Tutorials/DesignPatterns/banked_memory_system/README.md

```diff
@@ -0,0 +1,286 @@
+# `Banked Memory System` Sample
+
+This code sample demonstrates how to optimize a banked memory system.
+
+## Purpose
+
+When compiling your design for FPGA architectures, the Intel® oneAPI DPC++/C++ Compiler can choose to implement your variables as registers, or as block memories. The details of this decision are better defined in the [Memory System code sample](DirectProgramming/C++SYCL_FPGA/Tutorials/Features/memory_attributes). However, an ill-formed memory layout can lead to negative impact on your system performance. 
+
+This tutorial will teach you:
```

Comment:
Use simple present tense as much as possible. It's more direct and easier to read.

"This tutorial teaches you:"

Ideally, a list introduction should always be a complete sentence, but  that's not urgent to fix here.